src/main/drivers/rx/rx_pwm.c

```diff
@@ -409,6 +409,7 @@ void pwmRxInit(const pwmConfig_t *pwmConfig)
 
 void ppmAvoidPWMTimerClash(TIM_TypeDef *pwmTimer)
 {
+#ifdef USE_PWM_OUTPUT
```

Comment:
Good finding!

Unfortunately this will now throw a warning because `pwmTimer` is unused if `USE_PWM_OUTPUT` isn't defined. This should use the same pattern, and put the conditional around the entire function (and the location calling it).

Reply:
ahh yes of course.  missed that, fixed now.